gatsby-node.js

```diff
@@ -1,8 +1,70 @@
 /* eslint-disable no-unused-expressions */
 /* eslint-disable max-len */
 
-/* Create slug from file-path and append that slug to the 'fields' section under GraphQl
-so that we can reference it inside pages */
+// +------------------------------------------------------------------+
+// | Fetch remote data (contributors) at build-time                   |
+// | Later, maybe use the Github GraphQl API to source data in one go |
+// +------------------------------------------------------------------+
+const fetch = require('node-fetch');
+
+// https://www.gatsbyjs.org/docs/creating-a-source-plugin/#sourcing-data-and-creating-nodes
+// https://www.gatsbyjs.org/docs/data-fetching/
+exports.sourceNodes = async ({
+  actions: { createNode },
+  createNodeId,
+  createContentDigest,
+}) => {
+  // get data from GitHub API at build time
+  const result = await fetch('https://api.github.com/repos/fabricjs/fabric.js/contributors?per_page=10');
```

Comment:
We'd maybe later use the [Github GraphQl](https://developer.github.com/v4/) API to fetch all of the needed data in one go! Plus we only fetch the data we need!

Also, data caching could be implemented for development runs!